app/Models/Country.php

```diff
@@ -4,10 +4,53 @@
 
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
 
 class Country extends Model
 {
     use HasFactory;
 
     protected $fillable = ['name', 'short_code'];
+
+    /**
+     * Relationship with sports.
+     *
+     * @return BelongsToMany
+     */
+    public function sports(): BelongsToMany
+    {
+        return $this->belongsToMany(
+            Sport::class,
+            'countries_sports',
```

Comment:
@Evolyuta if you follow name convention and name the pivot table counttry_sport then you would not need to specify fields parameters